Computers/Computers/Program.cs

```diff
@@ -0,0 +1,278 @@
+            // Note: use loops and if-else statements
+            // Note: use the same loop for 6) and 7)
+            var desktopQuantity = 0;
+            var laptopQuantity = 0;
+            var serverQuantity = 0;
+            var computerQuantity = 0;
+            for (int i = 0; i < departments.Length; i++)
+            {
+                for (int j = 0; j < departments[i].Length; j++)
+                {
+                    computerQuantity += 1;
+                    switch (departments[i][j].ComputerType)
+                    {
+                        case ComputerTypeEnum.Desktop:
+                            desktopQuantity += 1;
+                            break;
+                        case ComputerTypeEnum.Laptop:
+                            laptopQuantity += 1;
+                            break;
+                        case ComputerTypeEnum.Server:
+                            serverQuantity += 1;
+                            break;
+                    }
+                }
+
+            }
+
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Desktop}s in departments is: {desktopQuantity}");
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Laptop}s in departments is: {laptopQuantity}");
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Server}s in departments is: {serverQuantity}");
+            Console.WriteLine($"The total number of all computers in departments is: {computerQuantity}");
+            Console.WriteLine();
+
+            // 8) find computer with the largest storage (HDD) - 
+            // compare HHD of every computer between each other; 
+            // find position of this computer in array (indexes)
+            // Note: use loops and if-else statements
+            var index1 = 0;
+            var index2 = 0;
+            var largestStorage = 0;
+            for (int i = 0; i < departments.Length; i++)
+            {
+                for (int j = 0; j < departments[i].Length; j++)
+                {
+                    if (departments[i][j].Hdd > largestStorage)
+                    {
+                        largestStorage = departments[i][j].Hdd;
+                        index1 = i;
+                        index2 = j;
+                    }
+                }
+            }
+
+            Console.WriteLine($"Computer with the largest storage is located in the array at positions {index1}, {index2}");
+            Console.WriteLine();
+
+            // 9) find computer with the lowest productivity (CPU and memory) - 
+            // compare CPU and memory of every computer between each other; 
+            // find position of this computer in array (indexes)
+            // Note: use loops and if-else statements
+            // Note: use logical oerators in statement conditions
+            index1 = 0;
+            index2 = 0;
+            var lowestProductivity = departments[0][0].CpuCores * departments[0][0].CpuFrequency;
```

Comment:
What if at position 0,0 would be computer with type server?

Reply:
then nothing will be changed, we just by an accidentally pick correct one, it will work, but yeah, this moment strange one

Reply:
> 

It will change to laptop, because it`s productivity is lower.

Reply:
Yes, it will change, but in this case. What would be if your first computer in department 1 is server, than computer with lowest productivity is computer with 8 cores and 3 GHz CPU frequency, in this case you will never found the correct one

Reply:
i don't like such moment as well, we don't need to guess, we have to know for sure)